source/CardGames.py

```diff
@@ -30,7 +30,7 @@ def __eq__(self, other):
       self.value == other.value
 #######################################################################################################################
 class Deck:
-  def __init__(self):
+  def __init__(self,deck_count=1):
```

Comment:
A brief description of the functionality might be helpful for later review.

Reply:
I have added a comment